src/reactviews/pages/QueryResult/resultGrid.tsx

```diff
@@ -92,6 +95,11 @@ const ResultGrid = forwardRef<ResultGridHandle, ResultGridProps>(
             setRefreshKey((prev) => prev + 1);
         };
         const resizeGrid = (width: number, height: number) => {
+            if (!table) {
+                context.log("resizeGrid - table is not initialized");
```

Comment:
Do you think console log is good enough here or do we want to use the new logger?

Reply:
Oops you are correct 😄